virtual_workstation_setup.sh

```diff
@@ -146,6 +146,46 @@ setup_master() {
     echo "Finished setting up the mongo repo..."
 }
 
+setup_50() {
+    if [[ -d mongo-v50 ]]; then
+        echo "'mongo-v50' dir exists; skipping setup"
+        return
+    fi
+
+    echo "Setting up the 5.0 branch..."
+    pushd "$workdir/mongo"
+        git worktree add "$workdir/mongo-v50" v5.0
+    popd
+
+    pushd "$workdir/mongo-v50"
+        mkdir -p src/mongo/db/modules
+        git clone git@github.com:10gen/mongo-enterprise-modules.git -b v5.0 src/mongo/db/modules/enterprise
+
+        /opt/mongodbtoolchain/v3/bin/python3 -m venv python3-venv
+
+        # virtualenv doesn't like nounset
+        set +o nounset
+        source python3-venv/bin/activate
+        set -o nounset
+
+            # The bundled pip version is very old (10.0.1), upgrade to the newest version that still
+            # uses the original resolver. See SERVER-53250 for more info.
+            python -m pip install --upgrade "pip<20.3"
```

Comment:
This has changed since 4.4. venv setup should run `python -m pip install "pip==21.0.1"`. You can also delete the comment just above this line.